### DISCUSSION TITLE
Event filtering patterns to streams event type on localstack

### DISCUSSION THREAD

**Discussion by duvansh91** [serverless/serverless#10407] [Q&A]
### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Use case description

Hi.
There is a recent release with a functionality to filter stream events: [https://www.serverless.com/framework/docs/providers/aws/events/streams/#setting-filter-patterns](https://www.serverless.com/framework/docs/providers/aws/events/streams/#setting-filter-patterns)

Does it work on localstack? I had been trying to use the `filterPatterns` tag but doesn't work for me. If it is not available for localstack, would be great to have it :raised_hands:.

**Comment by pgrzesik** [comment]
Hello @duvansh91 - this is not related directly to Framework Core in any way and is more of a Question than an Issue - I'm moving it to discussions.
